app/jinja_filters.py

```diff
@@ -751,10 +750,12 @@ def map_list_collector_config(
         if item_label:
             row_item["valueList"] = [{"text": item_name}]
 
-        row_item["title"] = item_label or item_name
-        row_items: list = [row_item]
+        title = item_label or item_name
 
         if related_answers:
+            # Supplementary Data (List Collector Content) list summaries don't use the row_item created above, as they are not editable
```

Comment:
Maybe something like this might be clearer?
```suggestion
            # List summaries driven by List Collector Content pages don't use the row_item created above, as they are not editable
```